### DISCUSSION TITLE
Any reason to contribute changes?

### DISCUSSION THREAD

**Discussion by kinoko87** [FunkinCrew/Funkin#2050] [Q&A] [answered]
Is there any reason one would make any changes and then create a pull request for Funkin, at the moment? Considering the amount of time it is taking for the update and no official changes from ninjamuffin & devs with him on the repo, wouldn't a majority of these changes be incompatible once the source code for the eventual update is pushed?

I am guessing that a lot of the code is going to be revamped in the update due to the game jam version nature of the repository code, but if I am wrong I guess this question is dismissible.

Thanks.

**Comment by EliteMasterEric** [comment, marked as answer]
Hey randomly just happened to see this.

Do NOT submit pull requests to the repository right now, they are not being reviewed or approved.

The game's code has been almost completely refactored. Any bugs you report or changes you suggest are likely either fixed or redundant.

We will be re-opening pull requests some time after the update is out, and we'll be more than happy to accept contributions then.

**Reply by kinoko87** [reply]
Thanks!
